emgapi/views.py

```diff
@@ -504,7 +505,11 @@ def retrieve(self, request, *args, **kwargs):
         ---
         `/super-studies/1`
```

Comment:
```suggestion
        `/super-studies/1`
        `/super-studies/tara`
```

Reply:
mhh I just want to add the example there, but I'm unsure if we should we `tara` or `human-micro` as in the test suite